core/src/main/java/com/google/cloud/sql/ConnectionConfig.java

```diff
@@ -134,6 +153,14 @@ public List<IpType> getIpTypes() {
     return ipTypes;
   }
 
+  public String getAdminRootUrl() {
+    return adminRootUrl;
+  }
+
+  public String getlAdminServicePath() {
```

Comment:
I think this is a typo. Maybe it should be `getAdminServicePath`

Reply:
Thank you for catching that. Updated.